pkg/resources/stream_acceptance_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 )
 
-func TestAcc_Stream(t *testing.T) {
+func TestAccStream(t *testing.T) {
```

Comment:
**[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT003: acceptance test function name should include underscore